Makefile

```diff
@@ -5,7 +5,7 @@ PKG := "github.com/Cybersecurity-and-Enterprise-Security/$(PROJECT_NAME)/cmd/$(P
 UNIT_TESTS = $(shell go list ./... | grep -v /e2e)
 
 MIN_GO_MAJOR_VERSION := 1
-MIN_GO_MINOR_VERSION := 22
+MIN_GO_MINOR_VERSION := 23
```

Comment:
This change is incomplete, there are more changes required to change the go version of the project.
If you revert this single line (and of course rename commit and MR), we could directly merge this :slightly_smiling_face: